security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/BaseBuilder.java

```diff
@@ -501,4 +503,13 @@ String scopeAudience() {
     String name() {
         return TenantConfigFinder.DEFAULT_TENANT_ID;
     }
+
+    boolean optionalAudience() {
+        return optionalAudience;
+    }
+
+    void setOptionalAudience(boolean optional) {
```

Comment:
we do not use verbs in setters, it should be just `optionalAudience(boolean)`, unless there is a conflict (which should not be)

Reply:
optionalAudience(boolean) is config blueprint method so I could not use the same name as setter. The only way how to fix this is to define blueprint method in the BaseBuilder.